README.md

```diff
@@ -27,6 +27,7 @@ You can find the description for the available service types and their configura
 ### 4.0.1
 
 + Updated `sailfish-core` version from `3.3.54` to `3.3.93`
++ New `fix` client property `encryptMethod`: password encryption support
```

Comment:
```suggestion
```
README in root should contain description for common changes for all sub-projects